### ISSUE TITLE
Temperature shows N/A on log viewer when it's actually got a value

### ISSUE THREAD

**Issue by ericmjl** [ericmjl/llamabot#115]
<img width="719" alt="image" src="https://github.com/user-attachments/assets/a6c0c735-dcfd-4be1-94eb-5d693328fca5">

See screenshot! This looks like a very subtle bug in the UI code.

**Comment by amirmolavi** [comment]
Hi Eric, Is it okay if I look into this? How can I reproduce this?
P.S. I started to look more in llamabot because I am trying to learn more about LLM and this is the way I learn, jump both feet in.

**Comment by ericmjl** [comment]
> Hi Eric, Is it okay if I look into this? 

Yes, please do so, @amirmolavi! I'd love to collaborate :smile:.

> How can I reproduce this? 

There should be a "getting started" guide on the README, which should boil down to the following commands: 

```bash
pixi install
pixi shell
llamabot visualize launch 
# this command name was given by an LLM and I really don't like it but it's stuck! 
# I think we should rename it to `llamabot logviewer launch` or something like that.
```

If the documentation feels sparse, please feel free to contribute as well -- I've had much success with drafting documentation for llamabot using `llamabot docs write`.

> P.S. I started to look more in llamabot because I am trying to learn more about LLM and this is the way I learn, jump both feet in.

🎉 that's amazing! Thanks for choosing LlamaBot as the place to do your learning :smile:. As you get more familiar, let's see how we can build better [cogs](https://www.dbreunig.com/2024/10/18/the-3-ai-use-cases-gods-interns-and-cogs.html) with LLMs!

**Comment by amirmolavi** [comment]
Hi Eric, I still have difficulties reproduce this. This is what I am getting now by running `llamabot log-viewer launch`: 
<img width="1085" alt="image" src="https://github.com/user-attachments/assets/c60484b4-99f1-4b6d-b99e-d0f8606c5d04">

**Comment by ericmjl** [comment]
@amirmolavi I just realized that some additional upgrades to the log-viewer UI broke the UI's ability to display logs. If you try now, does it work?
